java/test/src/main/java/io/ray/test/CrossLanguageInvocationTest.java

```diff
@@ -549,4 +549,20 @@ public void testPyCallJavaOverloadedMethodByParameterSize() {
             .remote();
     Assert.assertEquals(true, res.get());
   }
+
+  @Test
+  public void testCppActorMaxConcurrency() {
```

Comment:
Can we also add some tests to verify the concurrent calls any more? For example, you can submit only one task or submit 5 tasks(>3)?
And you can try to set setMaxConcurrency(1)?